search-app/src/cmr/search/services/parameters/parameter_validation.clj

```diff
@@ -280,6 +281,12 @@
                 (cpv/validate-date-time "revision-date end" end-date)))))
         revision-date))))
 
+(defn created-at-validation
+  [concept-type params]
+  "Validates that created-at parameter contains valid datetime strings."
+  (when-let [created-at (:created-at params)]
```

Comment:
I had started to pull out the common functionality from `revision-date-validation`, and when I saw that there was already a date-range validation below `cpv/validate-date-time`, I opted for the simpler solution. I can make this change fairly simply for the sake of consistency.